algebra/src/test/java/com/hubspot/algebra/ResultTest.java

```diff
@@ -1,16 +1,16 @@
 package com.hubspot.algebra;
 
-import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
I believe your import settings are still incorrect. Static imports should go below non-static ones by our standard style rules.

Reply:
Sorry about that, I can't get the HubSpot plugin to format this repo x.x